pkgs/shells/fish/dont-codesign-on-mac.diff

```diff
@@ -4,9 +4,9 @@ index 7d220a032..786b60e6e 100644
 +++ b/CMakeLists.txt
 @@ -183,7 +183,6 @@ ENDFUNCTION(CODESIGN_ON_MAC target)
  # Define a function to link dependencies.
- FUNCTION(FISH_LINK_DEPS_AND_SIGN target)
-   TARGET_LINK_LIBRARIES(${target} fishlib)
--  CODESIGN_ON_MAC(${target})
- ENDFUNCTION(FISH_LINK_DEPS_AND_SIGN)
+ function(FISH_LINK_DEPS_AND_SIGN target)
```

Comment:
This patch can be dropped and replaced by adding `-DMAC_CODESIGN_ID=OFF` to the `cmakeFlags` (see: https://github.com/fish-shell/fish-shell/issues/6952#issuecomment-620751620).

Reply:
Thanks! Deleted the patch and added the flag.